src/common/TagContainer/TagContainer.css

```diff
@@ -0,0 +1,46 @@
+.tag-container {
+    height: 70px;
+    border-radius: 18px;
+    margin: 2%;
+    display: flex;
+    flex-wrap: wrap;
+    overflow-y: scroll;
+}
+
+.tag {
```

Comment:
I made this comment here but applies to all the other .css as well.

Always try to add a custom classname to the main element (in this case I think is `'tag-container'`) and all the other elements inside that main element should be selected by a child or a descendant selector ([the main difference](https://www.geeksforgeeks.org/css-child-vs-descendant-selectors/)).

This way you ensure that the selector ONLY selects, for example, a `.tag` inside the `.tag-container` and NOT all the `.tag` anywhere on the app.